app.py

```diff
@@ -196,14 +106,20 @@ def process_liquidity(main_chart_data: pandas.DataFrame, collateral_token: str,
     # Process main chart data
     main_chart_data = main_chart_data.astype(float)
     debt_token_underlying_address = src.helpers.UNDERLYING_SYMBOLS_TO_UNDERLYING_ADDRESSES[debt_token]
-    main_chart_data = add_ekubo_liquidity(
+
+    ekubo_liquidity = EkuboLiquidity(
         data=main_chart_data,
         collateral_token=collateral_token_underlying_address,
         debt_token=debt_token_underlying_address,
+
+    )
+    main_chart_data = ekubo_liquidity.apply_liquidity_to_dataframe(
+        ekubo_liquidity.fetch_liquidity(),
```

Comment:
Wait, so this is now fetching only bids? Why not asks?

Reply:
As it was in add_ekubo_liquidity it's automatically getting or bids or asks.